pyo3/private/pyo3.bzl

```
@@ -208,13 +218,13 @@ def pyo3_extension(
         data = data,
         deps = [
             Label("//pyo3/private:current_rust_pyo3_toolchain"),
-            Label("@rules_python//python/cc:current_py_cc_libs"),
+            Label("//pyo3/private:py_headers_only"),
```

Why not pass `"@rules_python//python/cc:current_py_cc_headers"` here directly if it's the only thing `py_headers_only` is representing?
```suggestion
            Label("@rules_python//python/cc:current_py_cc_headers"),
```